drivers/ptx/src/main/java/uk/ac/manchester/tornado/drivers/ptx/mm/PTXKernelArgs.java

```diff
@@ -58,6 +57,45 @@ public List<CallArgument> getCallArguments() {
         return callArguments;
     }
 
+    public int getArgumentsTotalSizeInBytes() {
+        int size = 0;
+        for (CallArgument argument : callArguments) {
+            if (argument.isReferenceType()) {
+                size += 8; // Reference is 8 bytes
+            } else {
+                Class<?> klass = argument.getValue().getClass();
+                if (klass == Integer.class) {
+                    size += 4;
+                } else if (klass == Float.class) {
+                    size += 4;
+                } else if (klass == Short.class) {
+                    size += 2;
+                } else if (klass == Double.class) {
+                    size += 8;
+                } else if (klass == Long.class) {
+                    size += 8;
+                } else if (klass == Byte.class) {
+                    size += 1;
+                } else if (klass == int.class) {
+                    size += 4;
+                } else if (klass == float.class) {
+                    size += 4;
+                } else if (klass == short.class) {
+                    size += 2;
+                } else if (klass == double.class) {
+                    size += 8;
+                } else if (klass == long.class) {
+                    size += 8;
+                } else if (klass == byte.class) {
+                    size += 1;
+                } else if (klass == char.class) {
+                    size += 1;
+                }
```

Comment:
is there any probability to have `else`? have you covered all types?

Reply:
What do you mean?

Reply:
We have covered all primitives. This is detected as a changed because we moved code due to the code formatter. Not really a change.

Reply:
Sorry, as it is now, this code is never invoked. We can remove this function.